portfolio/src/main/webapp/about.js

```diff
@@ -1,82 +1,106 @@
-function loadInFact(index, fact_index, isTruth) {
-    let fact = isTruth ? truths[fact_index] : lies[fact_index];
+let correctAnswer = -1;
 
-    switch(index) {
-        case 0:
-            document.getElementById("first_txt").innerHTML = fact[0];
-            document.getElementById("first_img").src = fact[1];
-            break;
-        case 1:
-            document.getElementById("second_txt").innerHTML = fact[0];
-            document.getElementById("second_img").src = fact[1];
-            break;
-        case 2:
-            document.getElementById("third_txt").innerHTML = fact[0];
-            document.getElementById("third_img").src = fact[1];
-            break;
-    }
+/**
+ * Advances the game to the next question. Loads in the first question 
+ * game has not yet started. Otherwise, checks answer against user 
+ * selected anwer and alerts if a lie or truth has been chosen.
+ */
+function next()  {
```

Comment:
Apologies if this has already been addressed, since I'm having trouble finding it in the comment history and see you've added a comment to this method, but the name "next" seems kind of vague to me.  It makes me think "next what?"  Also, the style guide suggests that method names begin with a verb to make it clear that the method is *doing* something.  I'd suggest renaming this to advanceGame() because that definition seems to work for both cases of going from not started to started and going from one question to the next.

https://google.github.io/styleguide/jsguide.html#naming-method-names

Reply:
Changed